docs/connect.md

```diff
@@ -2,13 +2,26 @@
 
 To connect to the network with your own node, run through the followings steps.
 
+**Hardware Requirements:**
+**64bit** - please check before if your CPU is capable of running Node.js 8+ ([only supported by 64bit CPU](https://github.com/nodejs/build/issues/885) on Ubuntu machines)
+
+
 **Prerequisite:**
 
-- Node.js 8+
-- build-essential
+- Node.js ([Ubuntu / macOS](https://nodejs.org/en/download/) node.js 12+)
```

Comment:
not working on node 10 anymore? 🤔

Reply:
it is, but instead of confusing the user just let them download the latest node.js version (which seems to work now) otherwise he needs to look for older versions - just simpler user flow. 
- but we can add the info that it works from node.js 8+ on